crates/typos-dict/assets/words.csv

```diff
@@ -16600,6 +16754,7 @@ diverstiy,diversity
 divertion,diversion
 divertions,diversions
 divet,divot
+diviation,deviation
```

Comment:
Could you update this to `diviation,divination,deviation`

Reply:
Updated at [cb37366](https://github.com/crate-ci/typos/pull/530/commits/cb3736663e61ab7d97839481696bae0c77434865)